quarkchain/core.py

```diff
@@ -801,13 +801,13 @@ def finalize_merkle_root(self):
         self.meta.hash_merkle_root = self.calculate_merkle_root()
         return self
 
-    def finalize(self, evm_state, coinbase_amount, hash_prev_root_block=None):
+    def finalize(self, evm_state, coinbase_amount_map, hash_prev_root_block=None):
```

Comment:
`coinbase_amount_map`  here is of type `TokenBalanceMap`, while in `create_block_to_append` the arg with the same name is a optional dict? can we unify the argument style here?

Reply:
yes very confusing indeed, the latest version uses different names to hint types:
coinbase_amount_map: TokenBalanceMap
coinbase_tokens: Dict